cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -209,6 +209,11 @@ var {{bicepName .Name}}Env = map(filter({{bicepName .Name}}AppSettingsArray, i =
 
 module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
   name: '{{bicepName .Name}}'
+  dependsOn: [
+    {{- if .DbCosmosMongo}}
+    cosmos
+    {{- end}}
+  ]
```

Comment:
Without this, the cosmos may created after ACA, it will return error like "can not get mongodb-url using managed identity xxx". This error is not caused of authorization, it's caused by the secret not written to keyvault yet.

Reply:
I see. Can you look into if there's a way we can leverage [exportedSecrets](https://github.com/Azure/bicep-registry-modules/blob/main/avm/res/document-db/database-account/README.md#Outputs)?

Reply:
Updated.